bw2io/extractors/excel.py

```diff
@@ -5,17 +5,27 @@
 
 def get_cell_value_handle_error(cell):
     """
-    Get the value of a cell, handling error types.
+    Retrieve the value of a given cell and handle error types.
 
     Parameters
     ----------
-    cell : openpyxl.cell.cell.cell
-        The cell to retrieve the value from.
+    cell : openpyxl.cell.cell.Cell
+        The cell to get the value from.
 
-    Returns
-    -------
+    Returns:
```

Comment:
@Cerebrovinny, something here broke the correct rendering of the documentation. Maybe the "`:`"?

<img width="790" alt="Screenshot 2023-04-18 at 12 07 56" src="https://user-images.githubusercontent.com/23102087/232745048-4020c5ea-9b10-4a32-909b-a9384f6e26d2.png">

Please fix this in a new PR.

Reply:
fixed